common/DevHome.Common.csproj

```diff
@@ -43,7 +43,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.Windows.CsWinRT" Version="2.0.4" />
-    <PackageReference Include="Microsoft.Windows.DevHome.SDK" Version="0.500.484" />
+    <PackageReference Include="Microsoft.Windows.DevHome.SDK" Version="0.1399.520.1732" />

```

Comment:
The SDK change from this PR is temporary and the reference will be modified once [https://github.com/microsoft/devhome/pull/3206](url) is merged into main.

Reply:
The published SDK version is now being used.